src/legacy/core_plugins/kibana/public/dashboard/np_ready/legacy_app.js

```diff
@@ -141,7 +136,7 @@ export function initDashboardApp(app, deps) {
           });
         },
         resolve: {
-          dash: function($rootScope, $route, redirectWhenMissing, kbnUrl) {
+          dash: function($rootScope, $route, redirectWhenMissing, kbnUrl, history) {
             return ensureDefaultIndexPattern(deps.core, deps.npDataStart, $rootScope, kbnUrl).then(
               () => {
                 const savedObjectsClient = deps.savedObjectsClient;
```

Comment:
Just a question to understand: when/where is the search by title given by the URL used? is it a legacy kind of thing?

Reply:
to clarify: it the code that follows these lines

Reply:
This is a hidden feature - you have to add the title param to the URL manually, but if you do you can link to / bookmark a prefiltered dashboard listing page. If the search string perfectly matches the title of an index pattern, it even redirects directly. No idea when it got introduced, but there are some functional tests covering it: `test/functional/apps/dashboard/dashboard_listing.js`